test/unit/dom/dom.js

```diff
@@ -112,29 +250,156 @@ describe('skatejs-named-slots dom', function () {
 
       expect(slot.getAssignedNodes().length).to.equal(0, 'slot');
       expect(host.childNodes.length).to.equal(0, 'light');
-    });
 
-    it('replaceChild()', function () {
-      const light1 = document.createElement('div');
-      const light2 = document.createElement('div');
+      let div_local1 = document.createElement('h1');
+      let div_local2 = document.createElement('h2');
+      let div_local3 = document.createElement('h3');
+      let div_local4 = document.createElement('h4');
+      let div_local5 = document.createElement('h5');
```

Comment:
Are these the correct names? I would expect them to be camel cased, and they're not actually divs